content/docs/0.6.0/usecases/quality-gates/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Keptn Quality Gates only
+description: Describes how Keptn allows to use quality gates without deployment and testing features of Keptn.
+weight: 26
+keywords: [quality-gates]
+aliases:
+---
+
+Describes how Keptn allows to use quality gates without deployment and testing features of Keptn.
+
+## About this tutorial
+
+Let's say you want to use your existing tools for deploying and testing your applications - you can still use Keptn Quality Gates for extensive evaluation of Service Level Objectives (SLOs).
+
+*A brief recap of SLO and SLI:* A service level objective (SLO) is a target value or range of values for a service level that is measured by a service level indicator (SLI). An SLI is a carefully defined quantitative measure of some aspect of the level of service that is provided. By default, the following SLIs can be used for evaluation, inspired by the [Site Reliability Engineering](https://landing.google.com/sre/sre-book/chapters/service-level-objectives) book from Google:
+
+* *Response Time*: The time it takes for a service to execute and complete a task or how long it takes to return a response to a request.
+* *System Throughput*: The number of requests per second that have been processed.
+* *Error Rate*: The fraction of all received requests that produced an error.
+
+For more information about SLO and SLI, please take a look at [Specifications for Site Reliability Engineering with Keptn](https://github.com/keptn/spec/blob/0.1.1/sre.md).
+
+## Prerequisites
+
+* Running Keptn installation or a quality gates only installation as explained below
+
+* Clone example files used for this tutorial:
+
+    ```console
+    git clone --branch 0.6.0.beta https://github.com/keptn/examples.git --single-branch
+    ```
+
+    ```console
+    cd examples/onboarding-carts
+    ```
+
+## Install Keptn just for this use case
+
+If you want to install Keptn just to explore the capabilities of Keptn quality gates, you have the option to roll-out Keptn **without** components for automated delivery and operations. Therefore, set the `use-case` flag to `quality-gates` when executing the [install](../../reference/cli/#keptn-install) command as shown below:
+
+```console
+keptn install --platform=[aks|eks|gke|openshift|pks|kubernetes] --use-case=quality-gates
+```
+
+## Configure Keptn
+
+If you have completed another tutorial with the sockshop project and carts service, you can either delete the project using the [delete project](../../reference/cli/#keptn-delete-project) command or you use another project name throughout this tutorial, e.g., *rockshop*.
+
+* Create the Keptn project *sockshop* with only one the *hardening* stage declared in the `shipyard_quality_gates.yaml` file:
+
+  ```
+  keptn create project sockshop --shipyard=shipyard_quality_gates.yaml
+  ```
+
+* Create the Keptn service *carts* that you want to evaluate using Keptn quality gates:
```

Comment:
Please add a short note here stating that **creating a service** is not the same as **onboarding a service** and reference to the CLI docs.

Reply:
Good idea. done.